src/thunderbots/software/ai/hl/stp/tactic/shoot_goal_tactic.cpp

```diff
@@ -45,19 +52,19 @@ double ShootGoalTactic::calculateRobotCost(const Robot &robot, const World &worl
         // If we can intercept the ball, use the distance to the intercept point.
         // We normalize with the total field length so that robots that are within the
         // field have a cost less than 1
-        cost = (ball_intercept_opt->first - robot.position()).len() /
+        cost = std::pow((ball_intercept_opt->first - robot.position()).len(), 6) /
                world.field().totalLength();
     }
     else
     {
         // If we can't intercept the ball, just use the distance to the ball's current
         // position. We normalize with the total field length so that robots that are
         // within the field have a cost less than 1
-        cost = (world.ball().position() - robot.position()).len() /
+        cost = std::pow((world.ball().position() - robot.position()).len(), 6) /
                world.field().totalLength();
     }
 
-    return std::clamp<double>(cost, 0, 1);
```

Comment:
Should be in `[0,1]`, we should have a comment clearly explaining why we're doing this, and a link to a ticket to fix it.

Reply:
Same hack as above.

Reply:
fixed